### PR TITLE
hypershift: add 4.14 job

### PR DESCRIPTION
**Pull request by LiangquanLi930**
enable HyperShift CI(Critical) in 4.14

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision-chain.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ chain:
   as: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision
   steps:
   - chain: cucushift-installer-rehearse-aws-ipi-ovn-provision
-  - ref: hypershift-install
+  - chain: cucushift-hypershift-extended-install-private
```
Comment (heliubj18) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision-chain.yaml:5]: Will this change also apply to version 4.13?

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
/test build-clusters
--


<br class="Apple-interchange-newline">

**Pull request closed without merging**